### PR TITLE
Polish delete command

### LINKED ISSUE #108: Polish delete assignment
Currently, there is no way to handle ambiguous students names.

### PR DESCRIPTION
**Pull request by Edsel-Tan**
Closes #108

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteAssignmentCommand.java [hunk 1]
```diff
@@ -39,29 +45,58 @@ public class DeleteAssignmentCommand extends Command {
...
+        }
 
-        if (student == null) {
+        if (studentList.isEmpty()) {
             throw new CommandException(MESSAGE_NO_STUDENT_FOUND);
         }
 
-        Assignment assignment = student.deleteAssignment(assignmentQuery);
+        if (studentList.size() > 1) {
+            throw new CommandException(MESSAGE_DUPLICATE_STUDENT);
```
Comment (Ansel-Ch) [inline src/main/java/seedu/address/logic/commands/DeleteAssignmentCommand.java:95]: You have not implemented the part where you format the student numbers is it? Cause your String constant contains a placeholder for an argument.

**Review by Ansel-Ch — COMMENTED** [review]
Overall, LGTM. Just one comment about the Duplicate student string.

File: src/main/java/seedu/address/logic/commands/DeleteAssignmentCommand.java (hunk 1 above)
Comment (Edsel-Tan) [reply src/main/java/seedu/address/logic/commands/DeleteAssignmentCommand.java]: Fixed

File: src/main/java/seedu/address/model/Model.java [hunk 2]
```diff
@@ -132,6 +132,6 @@ public interface Model {
 
     void updateFilteredStudentList(Predicate<Student> predicate);
 
-
+    ObservableList<Student> getAllStudentByName(Name name);
```
Comment (natlv) [inline src/main/java/seedu/address/model/Model.java:135]: should we change this method to be called getAllStudentsByName?

File: src/main/java/seedu/address/logic/commands/DeleteAssignmentCommand.java [hunk 3]
```diff
@@ -26,6 +32,7 @@ public class DeleteAssignmentCommand extends Command {
             + "Parameters: "
             + PREFIX_NAME + "STUDENT_NAME "
             + PREFIX_ASSIGNMENT + "ASSIGNMENT "
+            + PREFIX_STUDENT_NUMBER + "STUDENT NUMBER (OPTIONAL)"
```
Comment (gnoossk) [inline src/main/java/seedu/address/logic/commands/DeleteAssignmentCommand.java:35]: image
Remember to add whitespaces in between the (OPTIONAL) and the next command

File: src/main/java/seedu/address/model/Model.java (hunk 2 above)
Comment (Edsel-Tan) [reply src/main/java/seedu/address/model/Model.java]: Fixed

**Review by gnoossk — APPROVED** [review]

**Pull request merged**